personas.md

```diff
@@ -0,0 +1,45 @@
+# OSAC Personas
+
+These are the essential personas that OSAC must target for an MVP. Many
+additional personas can be identified within the solution space, some of which
+will be added to this document over time as the project is ready to address
+their unique use cases.
+
+## Cloud Provider Admin
+
+* works for the Cloud Provider
+* handles or facilitates tenant onboarding
+* sets quotas for tenant organizations
+* uses the OSAC interfaces to manage tenant organizations
+* uses the OSAC interfaces to manage resource allocation
```

Comment:
In the past, we assumed that tooling for managing tenants and resource allocations would be external to the OSAC solution (the idea being that the cloud provider would already have facilities for managing these things, and our goal would be to integrate with them rather than replace them). This suggests we have expanded our scope to become a more "full service" solution. Is that an accurate interpretation?